### ISSUE TITLE
Code availability

### ISSUE THREAD

**Issue by seabbs** [ImperialCollegeLondon/covid19local#200]
Hi, this is a really nice piece of work. I really like the new (I think) history of projections plots. 

Could you point me in the direction of the modelling code to reproduce it? In your preprint I see links to the `epidemia` branch you used but I don't see a link to the analysis code? I'm sure this must be available for reviewers to use and I've just missed it so sorry to ping you with an issue! 

Thanks, 

Sam

**Comment by s-mishra** [comment]
Hi Sam,

I assume you are asking the code for latent factor models to reproduce the tiers estimates? I am in the process of uploading that code here https://github.com/ImperialCollegeLondon/covid19model. Should be up by tomorrow. 

If you need Rt estimates used in that work you can get that from here https://imperialcollegelondon.github.io/covid19local/#downloads

Now if you need the code that produces these Rt estimates (not part of tiers work as tiers work essentially grabs the Rt estimates from the above-listed location) then the whole machinery is a bit convoluted as we need to make sure everyday we successfully run it for all local areas in UK. Happy to discuss and pass the code we use to generate that bit too.

**Comment by seabbs** [comment]
Thanks @s-mishra, 

I've actually already emailed Daniel about the code for the latent factor analysis. 

Here I specifically meant the code for reproducing these Rt estimates as there is quite a bit more to it than just using `epidemia` off the shelf. Ideally along with the code used for pipelining and generating the website.

**Comment by s-mishra** [comment]
Yes @seabbs  you are right about that; especially the fact that the model use seroprevalence data too first to get a nationwide reliable estimate for IAR and IFR. We are pushing to get the whole code up in a state that is usable for everyone. Meanwhile, I will do share two basic scripts that are required to get the estimates for each ltla soonish (coming Friday/Sunday)

**Comment by seabbs** [comment]
Sounds great looking forward to it. 

I'd be very interested in hearing the rationale for not sharing previously in the original form given the potential impact on policy? Perhaps less required as I assume those using the downstream estimates did their own code review or you have some really robust internal processes.
